package/nfs/rancher-nfs

```diff
@@ -1,70 +1,70 @@
 #!/bin/bash
 
-# Notes:
-#  - Please install "jq" package before using this driver.
-
 if [ -e "$(dirname $0)/common.sh" ]; then
     source $(dirname $0)/common.sh
 elif [ -e "$(dirname $0)/../common/common.sh" ]; then
     source $(dirname $0)/../common/common.sh
 fi
 
-mount_default_share() {
-    local mntDest=$1
-
-    # get host:export from environment variables
-    if [ -z "${NFS_SERVER}" ]; then
-        print_error "Failure: No environment variable NFS_SERVER found\"}"
-    fi
-
-    if [ -z "${MOUNT_DIR}" ]; then
-        print_error "Failure: No environment variable MOUNT_DIR found\"}"
-    fi
+mount_nfs() {
+    local host=$1
+    local exportDir=$2
+    local mountDir=$3
+    local opts=$4
+    local error
 
-    local mntOptions=""
-    if [ ! -z "${MOUNT_OPTS}" ]; then
-        mntOptions="-o ${MOUNT_OPTS}"
-    fi
+    if [ "$(ismounted $mountDir)" == 0 ]; then
+        mkdir -p $mountDir
 
-    # mount remote share if not mounted already
-    if [ $(ismounted "${mntDest}") == 0 ] ; then
-        mkdir -p "${mntDest}"
         get_host_process_pid
-        error=`nsenter -t ${TARGET_PID} -n mount "${NFS_SERVER}":"${MOUNT_DIR}" "${mntDest}" 2>&1`
+        local cmd="nsenter -t ${TARGET_PID} -n mount"
+        if [ ! -z "$opts" ]; then
+            cmd="$cmd -o $opts"
+        fi
+        cmd="$cmd $host:$exportDir $mountDir"
+
+        error=`$cmd 2>&1`
         if [ $? -ne 0 ]; then
-            print_error "Failed mount ${mntOptions} ${NFS_SERVER}:${MOUNT_DIR} ${mntDest}: ${error}"
+            print_error "Failed $cmd"
         fi
     fi
 }
 
-unmount_default_share() {
-    local mntDest=$1
+mount_default() {
```

Comment:
Where does this get called and what is it used for? I couldnt find any references to it, but im not sure if it is being called in some magical frameworky way